wasm-bindgen-cli: 0.2.67 -> 0.2.69

rizary opened the pull request:
https://github.com/rustwasm/wasm-bindgen/releases/tag/0.2.69
https://github.com/rustwasm/wasm-bindgen/releases/tag/0.2.68

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

@zimbatm 

###### Motivation for this change
Update wasm-bindgen-cli to the newest release version

###### Things done
regenerate `Cargo.lock` and fix hash for `wasm-bindgen-cli` version `0.2.69`

add rizary as package maintainer.

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

zimbatm commented:
Result of `nixpkgs-review pr 107088` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>wasm-bindgen-cli</li>
  </ul>
</details>

zimbatm commented:
```
hash mismatch in fixed-output derivation '/nix/store/maysfdqshdfg79p4rizc3pkckdda80zk-wasm-bindgen-cli-0.2.69-vendor.tar.gz.drv':
  specified: sha256-KMX0fqSOEmd+BZCbDwcK9PqR51OifSaCJOsWs+g+Ak8=
     got:    sha256-5i7YsMEETMZijVNG6TqjHJ7SN60j7FOxrMGqwY6nLvM=
```

rizary committed:
wasm-bindgen-cli: 0.2.67 -> 0.2.69

https://github.com/rustwasm/wasm-bindgen/releases/tag/0.2.69
https://github.com/rustwasm/wasm-bindgen/releases/tag/0.2.68

zimbatm commented:
Result of `nixpkgs-review pr 107088` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>wasm-bindgen-cli</li>
  </ul>
</details>
